src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -26,11 +30,19 @@
 public class CommitInfoAnalyzer {
     public static final DateFormat GIT_STRICT_ISO_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
 
+    public static final String TAB_SPLITTER = "\t";
+    public static final String MOVED_FILE_INDICATION = "=> ";
+    public static final String BINARY_FILE_CONTRIBUTION = "-";
+    public static final int STAT_ADDITION_INDEX = 0;
```

Comment:
can these be private?

Reply:
The reason I did not place this as private is because I am utilising these constant fields in `TestUtil`. Should I put this as private and declare the constant again in `TestUtil`?

Reply:
Have set them to be private.